testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/observability/micrometer/MicrometerOtelIntegrationTestCase.java

```diff
@@ -35,6 +28,12 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.wildfly.test.integration.observability.container.OpenTelemetryCollectorContainer;
+import org.wildfly.test.integration.observability.container.PrometheusMetric;
+
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
```

Comment:
You'll have to remind me where that's documented. I had it set up in my IDE, but I guess those settings were lost somehow. That might be a nice check to add to checkstyle.

Reply:
1. static
2. all others
3. java
4. javax
5. jakarta

?

Reply:
@jasondlee It's

static
 
java
javax

all others

Reply:
jakarta is just another 'all others'.  Lots of files 'violate' that though because jakarta used to be javax and then got string replaced, so the jakarta stuff is mixed in with javax.

Reply:
OK. Thanks. IDE settings re-updated and sync'd to The Cloud. Imports reordered.